torch_geometric/explainability/algo/dummyexplainer.py

```diff
@@ -0,0 +1,39 @@
+import torch
+
+from torch_geometric.explainability.explanations import Explanation
+
+from .base import ExplainerAlgorithm
+
+
+class RandomExplainer(ExplainerAlgorithm):
```

Comment:
Is this for testing purpose?

Reply:
For testing and benchmarking: if someone proposes a new explaination algorithms it can be compared to it (in the same spirit as a dummy model from sklearn).